README.md

```diff
@@ -1,31 +1,59 @@
 [![Slack Status](https://slack.projectcalico.org/badge.svg)](https://slack.projectcalico.org)
 [![IRC Channel](https://img.shields.io/badge/irc-%23calico-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#calico)
 [![Docker Pulls](https://img.shields.io/docker/pulls/calico/node.svg)](https://hub.docker.com/r/calico/node/)
-[![](https://badge.imagelayers.io/calico/node:latest.svg)](https://imagelayers.io/?images=calico/node:latest)
 
 # Calico
 <img src="http://docs.projectcalico.org/images/felix.png" width="100" height="100">
 
-Calico is an open source system enabling cloud native application connectivity and policy. Calico integrates with major orchestration
-systems like [Kubernetes](https://kubernetes.io), [Apache Mesos](http://mesos.apache.org/), [Docker](https://www.docker.com/), [OpenStack](https://www.openstack.org/) and more to provide a seamless
-experience for developers and operators.
+Calico is an open source networking and network security solution for containers, virtual machines, and bare-metal workloads.
+Calico uses standard Linux networking tools to provide two major services for Cloud Native applications:
 
-Calico is a [Tigera](https://www.tigera.io/) open source project, and is primarily maintained by the Tigera team.
+- Network connectivity between workloads.
+- Network security policy enforcement between workloads.
 
-## Get Started Using Calico
+Calico’s flexible architecture supports a wide range of deployment options, using modular components, including:
 
-For users who want to learn more about the project or get started with Calico, see the documentation on [docs.projectcalico.org](https://docs.projectcalico.org).
+- [CNI](https://github.com/projectcalico/cni-plugin) plugins for Kubernetes to provide highly efficient pod networking and IP
+  Address Management (IPAM).
+- A [Neutron ML2][(https://github.com/projectcalico/networking-calico) plugin to provide VM networking for OpenStack.
```

Comment:
I think you've got an extra `[` bracket here which is causing this link to fail to render.

Reply:
Fixed